.env.local

```diff
@@ -10,4 +10,6 @@ TYPEORM_LOGGING = true
 
 # API URL variables
 GEOCODER_API_BASE_URL= 'testurl'
-GEOCODER_API_ADDRESSES_ENDPOINT='endpoint_name.json'
\ No newline at end of file
+GEOCODER_API_ADDRESSES_ENDPOINT='endpoint_name.json'
+BCGEOCODER_API_KEY_PID = 'apikey'
+BCGEOCODER_TEST_API_URL_PID = 'pid testurl'
```

Comment:
Is this the test url combined with the endpoint name?

Reply:
Yeah they are combined, but good point I'll split that up